docs/development/intro.rst

```diff
@@ -171,7 +171,7 @@ To *activate* your development version (branch) of Gammapy in your environment:
 
 .. code-block:: bash
 
-    pip install -e .
+    python -m pip install -e .
```

Comment:
Why is this changed?

Reply:
It's just in general the recommended way to install into a specific environment. `pip` can in general live in a different environment than the `python` command. See e.g. http://jakevdp.github.io/blog/2017/12/05/installing-python-packages-from-jupyter/#How-to-use-Pip-from-the-Jupyter-Notebook